geode-assembly/src/test/java/org/apache/geode/rest/internal/web/SwaggerVerificationTest.java

```diff
@@ -42,6 +44,7 @@
   static Properties properties = new Properties() {
     {
       setProperty(START_DEV_REST_API, "true");
+      setProperty(SECURITY_MANAGER, SampleSecurityManager.class.getName());
```

Comment:
using SamplesecurityManager will require you to use a json file as well. Use SimpleSecurityManager instead.